console/my-account/troubleshooting/cant-log-into-my-account.mdx

```diff
@@ -0,0 +1,53 @@
+---
+meta:
+  title: I cannot log into my account
+  description: This page helps you log into your account
+content:
+  h1: I cannot log into my account
+  paragraph: This page helps you log into your account
+dates:
+  validation: 2022-10-19
+  posted: 2022-10-19
+---
+
+Sometimes you may have problems when connecting to your Scaleway account. If you have encountered any of the following possible cases, check out our solutions.
+
+## Unrecognized IP address error 
+You may try to log into your Scaleway account and see a message like the following: 
+
+<Lightbox src="scaleway-unrecognized-ip.webp" alt="" />
+
+This means you tried to log into your account from an unknown IP address. As a security measure, we require you to use a magic link to complete authentification. 
```

Comment:
Since this doc is aimed at people who might not be that tech-savvy, I wonder if we should add something like  "This suggests you might be trying to log in from a new device or location. As a security measure..."